sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/CreateEcKeyOptions.java

```diff
@@ -101,32 +106,64 @@ public CreateEcKeyOptions setTags(Map<String, String> tags) {
     }
 
     /**
-     * Set the enabled value.
+     * Set a value that indicates if the key is enabled.
+     *
+     * @param enabled The enabled value to set.
      *
-     * @param enabled The enabled value to set
-     * @return the EcKeyCreateOptions object itself.
+     * @return The {@link CreateEcKeyOptions} object.
      */
     @Override
     public CreateEcKeyOptions setEnabled(Boolean enabled) {
         super.setEnabled(enabled);
+
+        return this;
+    }
+
+    /**
+     * Set a value that indicates if the private key can be exported.
+     *
+     * @param exportable The exportable value to set.
+     *
+     * @return The updated {@link CreateKeyOptions} object.
+     */
+    public CreateKeyOptions setExportable(Boolean exportable) {
+        super.setExportable(exportable);
+
+        return this;
+    }
+
+    /**
+     * Set the policy rules under which the key can be exported.
+     *
+     * @param releasePolicy The release policy to set.
+     *
+     * @return The updated {@link CreateKeyOptions} object.
+     */
+    public CreateKeyOptions setReleasePolicy(KeyReleasePolicy releasePolicy) {
```

Comment:
This should not be included for 7.2. It's new for 7.3-preview.

Reply:
I will remove all traces of Secure Key Release in this PR as well then.